Cargo.toml

```diff
@@ -15,14 +15,15 @@ anyhow = "1.0"
 clap = "2.33.3"
 directories = "3.0.2"
 k8s-openapi = { version = "0.11.0", default-features = false, features = ["v1_20"] }
-policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", tag = "v0.1.7" }
+policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", branch = "metadata" }
```

Comment:
To be udpated before merge, blocking on https://github.com/kubewarden/policy-evaluator/pull/6